src/core/federation.ts

```diff
@@ -1178,6 +1254,25 @@ export class FederatedApp extends App {
         this.rtiClient.sendRTIStopRequestReply(tag);
     }
 
+    /**
+     * Send a port absent message to federate with fed_ID, informing the
+     * remote federate that the current federate will not produce an event
+     * on this network port at the current logical time.
+     * 
+     * @param additional_delay The offset applied to the timestamp
+     *  using after. The additional delay will be greater or equal to zero
+     *  if an after is used on the connection. If no after is given in the
+     *  program, NEVER is passed.
```

Comment:
`If no after is given in the program, NEVER is passed` -> This statement doesn't seem true from the implementation. Maybe we should update the implementation to match this description?